in_toto/common_args.py

```diff
@@ -163,6 +163,18 @@ def parse_password_and_prompt_args(args):
     ),
 }
 
+PKCS8_ARGS = ["--pkcs8"]
```

Comment:
`pkcs8` is a bit technical. We could also call the arg:
- private-key
- signing-key
- key-file
- key-path
- ...


Any thoughts?

Reply:
Fair. Let's go with signing-key then.